### DISCUSSION TITLE
캐릭터 아이디어

### DISCUSSION THREAD

**Discussion by seong0929** [seong0929/Platypus#54] [Ideas]
# 배경
사바나, 사막(흙의 기운)
바다, 강(물의 기운)
숲, 우림(풀의 기운)

# 캐릭터 분류
- 딜러
- 탱커
- 서포터

**Comment by seong0929** [comment]
예) SenorrZoro(바다, 딜러): 오리너구리가 펜싱칼을 들고 싸우는 캐릭터

**Comment by KimDa99** [comment]
사바나 (건조한 친구들, 모래의 기운)
1. 낙타(탱커)
2. 퉷스터(딜러)
3. 사막 여우(서포터)
4. 고릴라(탱커)
5. 고슴도치(딜러)

바다 (촉촉한 친구들, 물의 기운)
1. 상엉양(딜러)
2. 오리너구리(딜러)
6. 두꺼비(서포터)
7. 서핑 히피 강아지(탱커)
8. 마라카스 꽃게(서포터)

숲 (풀의 기운)
1. 공룡(탱커)
2. 양(서포터)
3. 반달가슴 곰(탱커)
4. 절구 토끼(딜러)
5. 어깨걸이 극라조(서포터)

15개 중 5탱커, 5딜러, 5서포터

**Comment by KimDa99** [comment]
"//"이 있으면 후보, 없으면 확정되어 있는 캐릭터

**사바나 (건조한 친구들, 모래의 기운)**
1. 낙타
2.  퉷스터
3. 사막 여우
4. 고릴라
5. 고슴도치
// 사막 도마뱀 - 그림자를 좋아하는 히키코모리 사막 도마뱀 (암살자)
// 불 코끼리
// 혼란유발 얼룩말
// 그랩 거미

**바다 (촉촉한 친구들, 물의 기운)**
1. 상엉양
2. 오리너구리
3. 두꺼비
6. 서핑 히피 강아지 
7. 마라카스 꽃게
// 낚시 고양이 
// 빨판 문어

**숲 ( 풀의 기운)**
1. 공룡
2. 양
3. 반달가슴 곰
4. 절구 토끼
5. 어깨걸이 극락조
// 소음공격 고라니 
// 쓰레기통 너구리

**Reply by seong0929** [reply]
반달 가슴곰(숲, 탱커): 가슴에 있는 반달로 부메랑을 날림

기본 공격: 손으로 때리기
스킬: 반달 부메랑
궁: 반달->보름달 광폭화

**Reply by seong0929** [reply]
SenorrZoro(바다, 딜러): 오리너구리가 펜싱칼을 들고 싸우는 캐릭터

기본 공격: 펜싱 칼로 찌르기
스킬: 거리 벌리거나 좁히기
궁: 빠르게 한 적을 관통하기

**Reply by KimDa99** [reply]
퉷스터(사바나, 딜러): 다람쥐 입에 있는 해바라기 씨앗으로 공격

기본 공격: 씨앗 뱉기
스킬: 꼬리치기(상대 넉백 효과)
궁: 공중포격(하늘로 날라가 공중에서 포격한다)

**Reply by KimDa99** [reply]
항아리 두꺼비 (바다, 지원가): 항아리에서 새는 물로 팀원/적에게 효과를 주는 범위 서포터

패시브: 새는 물로 팀원 버프
기본 공격 없음, 이동만 함
스킬: 혀로 때리기
궁: 항아리가 깨지면서 큰 범위로 팀원 버프, 상대 디버프

**Reply by KimDa99** [reply]
초딩 공룡(숲, 탱커): 방어력이 높은 도발 공룡 탱커

기본 공격:  발 구르기
스킬:  메롱(도발 후, 알에 숨는다)
궁:  엄마! (알에 숨은 후, 엄마 공룡에게 전화해 엄마를 부른다)

**Reply by KimDa99** [reply]
제물(숲, 지원가): 양의 형상을 뛴 높은 체력을 가진 흡혈/지원가, 성스러운 양으로 보이지만, 사실을 제물로 바쳐졌던 염소

기본 공격: 기도 (적의 피를 흡혈한다.)
스킬: 천사 소환 (천사들은 팀원들을 치료해주는 천사를 자신의 피로 소환한다)
궁: 악마 소환 (광폭화, 주변에 있는 적의 피를 흡혈한다.)

**Reply by KimDa99** [reply]
농구 낙타(사바나, 탱): 쌍봉 낙타, 등에 있는 혹을 떼어 공격/자힐 한다.

기본 공격: 리바운드(첫번째 혹을 떼어 농구공 처럼 상대에게 던진다.)
스킬: 비상식량(두번째 혹을 흡수하여 자힐한다.)
궁:  슬램덩크(혹으로 위에서 아래로 찍어 누른다, 일정 범위 스턴)

**Reply by KimDa99** [reply]
상어양(바다, 딜러): 상어처럼 보이고 싶은 등 푸른 생선. 위급 시 등에 있는 등지느러미를 떼어서 공격한다.

기본 공격: 물어뜯기
스킬: 위협(이빨을 드러내어 위협, 디버프/스턴을 건다)
궁:  부메랑 (등에 있는 등지느러미를 떼어 공격한다. 강력한 직선 공격)

![image](https://github.com/seong0929/Platypus/assets/91198933/84e5fb2e-d01a-46d0-8d0f-18ded1eaafe2)

**Reply by KimDa99** [reply]
절구 토끼(숲, 딜러):

기본 공격: 떡 던지기 (원거리 공격)
스킬: 절구로 치기 (스턴기)
궁: 아마겟돈 (운석 낙하)

**Reply by KimDa99** [reply]
서핑 히피 강아지(바다, 탱커)

기본 공격: 보드로 때리기
스킬: 서핑 보드로 막기 - 일부 데미지 반사
궁: 파도 일으키기 ( 일정 범위에 파도를 일으킴 - 스턴 (나미 궁))

**Reply by KimDa99** [reply]
마라카스 꽃게 (바다, 지원가): 멕시코산으로 마라카스를 들고 있는 꽃게

기반 공격: 마라카스로 때리기
스킬: 마라카스 (연주, 힐)
궁: 탱고 (춤추면서 연주, 버프)

**Reply by KimDa99** [reply]
보석 사막여우 (사바나, 지원가): 머리에 보석이 박혀있는 고귀한 사막여우

기본 공격: 반짝거리기 ( (금융치료) 보석에서 빛이 나와 아군을 치료한다.)
스킬: 귀여운 척 하기 (상대 캐릭터 지정해서 공격력 낯추기)
궁: 벌크업 (공포)

**Reply by KimDa99** [reply]
카사노바 극락조 (숲, 서포터)

기본 공격: 깃털 날리기 
스킬: 구애 (매혹)
궁: 실연 (상대 디버프)

**Comment by KimDa99** [comment]
[GliderSprite.zip](https://github.com/seong0929/Platypus/files/11997072/GliderSprite.zip)

---
![GliderTotal](https://github.com/seong0929/Platypus/assets/91198933/268d6f78-4fb1-4eb2-9a25-8964eed5c90a)
![Seed](https://github.com/seong0929/Platypus/assets/91198933/d6eeddfe-35e6-4d57-8e22-4cd01fc5852f)

**Reply by seong0929** [reply]
### 설명
- IDLE
- Move
- Skill
- Attack
- Ult1
- Ult2

### 부족한 모션
- 데스
- 리스폰

**Comment by KimDa99** [comment]
![ToadSkill](https://github.com/seong0929/Platypus/assets/91198933/ec14f17d-546d-430e-b787-4fc5e56acc09)
![ToadULT](https://github.com/seong0929/Platypus/assets/91198933/c5c33a85-b74b-4e7c-9c94-386cf1836cbd)
![ToadIdle](https://github.com/seong0929/Platypus/assets/91198933/0d570f13-7cf5-48d5-9f4e-8f0c76d4034c)
![ToadMove](https://github.com/seong0929/Platypus/assets/91198933/6a7a41c6-34e9-41a4-9750-6d5cb2f0e77a)

**Reply by KimDa99** [reply]
기본 패시브: 범위 안에 있는 아군의 체력을 회복 시키며 이속을 빠르게한다.
1. Skil : 혀로 상대를 쳐 이속을 느리게하고 약간의 데미지를 준다
2. ULT: 항아리 안으로 들어가 물을 쏟는다. 큰 범위 안에 있는 아군의 체력을 회복 + 이속 크게 증가 + 상대 이속 크게 하락
3. Idle
4. Moving

**Reply by seong0929** [reply]
1. 기본 패시브도 노드에 추가해야하나? 여태 패시브 관련한 캐릭이 없었습니다.
2. 범위는 나 알아서?
3. 회복, 이송 증가 이팩트는... 일단 생략할게요
4. skill이 저 혀 길이 만큼...?
5. 근거리 캐릭터죠?

**Reply by seong0929** [reply]
PoToad
1. 기본 패시브 보류
2. 범위 임시
3. 임팩트 생략
4. 네
5. 네

**Reply by KimDa99** [reply]
![ToadTotal](https://github.com/seong0929/Platypus/assets/91198933/6bcbdc04-2bde-40c5-912e-698d8e827fa6)

**Reply by KimDa99** [reply]
![ToadSkill-separated](https://github.com/seong0929/Platypus/assets/91198933/9df02343-73a2-4306-aef5-8109a751dae6)
